test/orbits/keplerian.jl

```diff
@@ -0,0 +1,192 @@
+using Conda
+using PyCall
+using Transits.Orbits: KeplerianOrbit, relative_position
+using Unitful, UnitfulAstro
+
+Conda.add("batman-package"; channel="conda-forge")
+
+# Common Python functionality
+py"""
+import numpy as np
+from batman import _rsky
+"""
+const allclose = py"np.allclose"
+
+# Common Julia functionality
+function compute_r(orbit, t)
+    pos = relative_position.(orbit, t)
+    r = map(pos) do arr
+        √(arr[1]^2 + arr[2]^2)
```

Comment:
For the record, this operation is `hypot` (or also `norm` if you apply it on an array and load `LinearAlgebra`)

Reply:
Oh neat, I didn't know about `hypot`!

Reply:
175a0a1